packages/web-components/src/components/tabs-extended/__stories__/README.stories.mdx

```diff
@@ -20,6 +20,7 @@ Here's a quick example to get you started.
 
 ```javascript
 import '@carbon/ibmdotcom-web-components/es/components/tabs-extended/tabs-extended.js';
+import '@carbon/ibmdotcom-web-components/es/components/tabs-extended/tabs-extended-media.js';
```

Comment:
I'm wondering if there should be two separate sections in the documentation for just `dds-tabs-extended` and for `dds-tabs-extended-media`. If an adopter only wants `dds-tabs-extended`, they won't have any need for the extended with media version so wouldn't be something they need to import.